yarn.lock

```diff
@@ -7778,7 +7810,7 @@ file-loader@3.0.1, file-loader@^3.0.1:
     loader-utils "^1.0.2"
     schema-utils "^1.0.0"
 
-file-saver@eligrey/FileSaver.js#1.3.8:
+"file-saver@github:eligrey/FileSaver.js#1.3.8":
```

Comment:
@abefernan the CI [just got a new check](https://github.com/iov-one/ponferrada/pull/137) to reject this diff. The issue is: after `yarn upgrade`, this is `"file-saver@github:eligrey/FileSaver.js#1.3.8":` but after `yarn install`, this is changed to `file-saver@eligrey/FileSaver.js#1.3.8:`. At the end of the day, this is an annoying yarn bug and nobody's fault.

To avoid this CI fail, use the following strategy:
1. If not necessary, do not `yarn upgrade`
2. If you really need to upgrade something, bump the version of the dependency in the package.json and then run `yarn install`

Reply:
Ok, thank you. Not sure how that ended up there if I never performed a `yarn upgrade`.